DMX.Core.Api.Infrastructure.Provision/Brokers/Clouds/CloudBroker.WebApps.cs

```diff
@@ -19,18 +19,26 @@ public partial class CloudBroker
             IAppServicePlan appServicePlan,
             IResourceGroup resourceGroup)
         {
-            IExistingWindowsPlanWithGroup webAppWithExistingWindowsPlan = this.azure.AppServices.WebApps
-                .Define(webAppName)
-                    .WithExistingWindowsPlan(appServicePlan);
+            var webAppSettings= new Dictionary<string, string>
+                {
+                    { "ApiConfigurations:Url", this.externalLabApiUrl },
+                    { "ApiConfigurations:AccessKey", this.externalLabApiAccessKey }
+                };
 
-            IWithCreate<IWebApp> webAppWithConnectionString = webAppWithExistingWindowsPlan.WithExistingResourceGroup(resourceGroup)
-                .WithNetFrameworkVersion(NetFrameworkVersion.Parse("v7.0"))
+            IWithWindowsRuntimeStack webAppWithPlanAndResouceGroup =
+                this.azure.AppServices.WebApps
+                    .Define(webAppName)
+                        .WithExistingWindowsPlan(appServicePlan)
+                            .WithExistingResourceGroup(resourceGroup);
+            
+            return await webAppWithPlanAndResouceGroup
+                .WithNetFrameworkVersion(NetFrameworkVersion.Parse("v6.0"))
                     .WithConnectionString(
                         name: "DefaultConnection",
                         value: connectionString,
-                        type: ConnectionStringType.SQLAzure);
-
-            return await webAppWithConnectionString.CreateAsync();
+                        type: ConnectionStringType.SQLAzure)
+                        .WithAppSettings(settings: webAppSettings)
```

Comment:
Should .WithAppSettings be tabbed in one more time?

Reply:
That's right - corrected.